libs/components/src/lib/input-slider/stories/vl-input-slider.stories-doc.mdx

```diff
@@ -0,0 +1,26 @@
+import { ArgsTable, DocsStory, PRIMARY_STORY } from '@storybook/addon-docs';
+
+# Input Slider
+
+Gebruik een input-slider om een waarde te selecteren tussen een minimum en maximum aan de hand van een slider.
+
+## Voorbeeld
+
+```js
+import { VlInputSliderComponent } from '@domg-wc/components';
+```
+
+```html
+<vl-input-slider></vl-input-slider>
+```
+
+<DocsStory id="components-input-slider--input-slider-default" />
+
+## Configuratie
+
+<ArgsTable story={PRIMARY_STORY} />
+
+## Referenties
+
+Dit is een nieuw component binnen onze bibliotheek en is gebaseerd op de standaard HTML5 input range slider:  
```

Comment:
is het nuttig eventueel te vermelden dat de stijl gebaseerd is op input-range component van Digitaal Vlaanderen & eventueel ook uitlichten dat om verwarring te vermijden, dit niet werkt zoals de range (maar dat is mss overbodig)

Reply:
Done